assets/sass/components/setup/_googlesitekit-setup.scss

```diff
@@ -114,6 +114,26 @@
 		padding-left: 10px;
 		padding-right: 10px;
 	}
+
+	.googlesitekit-optimize-setup__form {
```

Comment:
I think these styles may not actually be necessary. Most of the styling appears to be fixed by wrapping the optimize field with a `div`.